db/migrate/20151106075351_add_school_id_to_student.rb

```diff
@@ -0,0 +1,5 @@
+class AddSchoolIdToStudent < ActiveRecord::Migration
+  def change
+    add_column :students, :school_id, :integer, null: false
```

Comment:
Want to add an `index` here since it seems like we're going to have indices for all associations?